phoenix-core/src/it/java/org/apache/phoenix/end2end/index/BaseIndexIT.java

```diff
@@ -1514,7 +1514,6 @@ public void testLastDDLTimestampOnAsyncIndexes() throws Exception {
 
             // run the index MR job.
             IndexToolIT.runIndexTool(false, TestUtil.DEFAULT_SCHEMA_NAME, tableName, indexName);
-            TestUtil.waitForIndexState(conn, fullIndexName, PIndexState.ACTIVE);
```

Comment:
@haridsv Would you remember why this change was made in this PR? This breaks a few tests on my PR https://github.com/apache/phoenix/pull/1883. 
Rushabh made the [change](https://github.com/apache/phoenix/pull/1595/files#diff-6d91af246300bfabfe5d43b94abe177fa549c3dc60d2e25e795e1425a0ff4bb4R1466) originally and I think it is needed. Please let me know if there was a reason to remove it in this PR.

Reply:
Actually, this change is causing these 3 tests to fail continuously on all PRs on master:
`GlobalImmutableTxIndexWithRegionMovesIT.testLastDDLTimestampOnAsyncIndexes`
`GlobalMutableTxIndexIT.testLastDDLTimestampOnAsyncIndexes`
`GlobalImmutableTxIndexIT.testLastDDLTimestampOnAsyncIndexes`

Reply:
I see it was removed based on Kadir's comment here - https://github.com/apache/phoenix/pull/1866#discussion_r1561771543

Reply:
@tkhurana Should I add it back to BaseIndexIT and BaseIndexWithRegionMovesIT in my PR? Not sure why the behaviour of IndexTool is different in these 3 cases.

Reply:
We will have to put this back, we are just discussing this internally.